app/views/handlers/primary_person_list_collector.py

```diff
@@ -31,11 +32,20 @@ def handle_post(self):
             self._primary_person_id = (
                 self.questionnaire_store_updater.add_primary_person(list_name)
             )
+
+            self.questionnaire_store_updater.update_same_name_items(
+                list_name, same_name_answer_ids
+            )
```

Comment:
I don't think this is needed here as the name is not entered until the next page, so we wouldn't know if it was a duplicate or not.